x-pack/plugins/infra/public/pages/error.tsx

```diff
@@ -44,13 +45,23 @@ export const ErrorPageBody: React.SFC<{ message: string }> = ({ message }) => {
         <EuiPageHeader>
           <EuiPageHeaderSection>
             <EuiTitle size="m">
-              <h1>Oops!</h1>
+              <h1>
+                <FormattedMessage
+                  id="xpack.infra.homePageError.oopsErrorTitle"
```

Comment:
it is errorPage, not homePageError

Reply:
also @maryia-lapata  maybe you know better name instead of **oopsErrorTitle**?

Reply:
maybe `unexpectedErrorTitle`